packages/light-client/src/rpc.ts

```diff
@@ -36,6 +44,14 @@ export class LightClientRPC {
     return utils.deepCamel(await request(this.uri, "get_tip_header"));
   }
 
+  async getPeers(): Promise<Array<RemoteNode>> {
+    return utils.deepCamel(await request(this.uri, "get_peers"));
```

Comment:
The `deepCamel` is not transparent enough, making it difficult to debug when something is changed or doesn't match the RPC. Can you refactor them using the param formatters and result formatters?

https://github.com/ckb-js/lumos/blob/cbab8d9882e52e730dba8c3c362c9ffce9b81b78/packages/rpc/src/Base/net.ts#L14